org.lflang.diagram/src/org/lflang/diagram/lsp/LanguageDiagramServer.java

```diff
@@ -30,10 +36,14 @@ private static class LFLsCreator extends AbstractLsCreator {
 
         @Override
         public Module createLSModules(boolean socket) {
-            return Modules.override(super.createLSModules(socket)).with(new AbstractGenericModule() {
+            return Modules2.mixin(Modules.override(super.createLSModules(socket)).with(new AbstractGenericModule() {
                 public Class<? extends LanguageServerImpl> bindLanguageServerImpl() {
                     return LFLanguageServer.class;
                 }
+            }), it -> {
+                // Temporary fix for an issue of Klighd with Xtext 2.28 (https://github.com/kieler/KLighD/issues/144)
```

Comment:
We should probably create and issue to track this.

Reply:
Done. #1473